src/lib/Behat/Component/Pagination.php

```diff
@@ -22,11 +22,13 @@ public function clickNextButton(): void
     {
         $currentPage = (int) $this->getHTMLPage()->find($this->getLocator('currentPage'))->getText();
         // scroll to the bottom to avoid "Go to top" button
-        $this->getHTMLPage()->executeJavaScript("document.querySelector('.ibexa-back-to-top-scroll-container').scrollTo(0, document.querySelector('.ibexa-back-to-top-scroll-container').scrollHeight)");
+        $this->getHTMLPage()->executeJavaScript("
+    var el = document.querySelector('.ibexa-back-to-top-scroll-container');
+    if (el) el.scrollTo(0, el.scrollHeight);
+    ");
```

Comment:
You can https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc to improve readability